### PR TITLE
Improve docs slightly

### PR DESCRIPTION
**Pull request by nchelluri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,9 +96,9 @@ want to limit this to just one.
 The `--project` argument will be matched case insensitively against all PT
 projects accessible to you.
 
-You can also omit this parameter and specify the project name via the
+You can also omit this option and specify the project name via the
```
Comment (autarch) [inline README.md:99]: This should be "argument" to be consistent with the previous paragraph.

**Review by autarch — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nchelluri) [reply README.md:99]: I was torn about this. The heading calls them command line options. In my opinion, the option is `--project "Project Name"` and the argument is `"Project Name"` so I think we should use argument here. An alternative is to use the same word throughout.

**Comment by oschwald** [comment]
@nchelluri, should we update, merge, or close this PR?

**Comment by nchelluri** [comment]
I'd say we should close this PR.

**Pull request closed without merging**